app/dojo.py

```diff
@@ -0,0 +1,34 @@
+from app.person import Fellow, Staff
+from app.room import LivingSpace, Office, Room
+
+
+class Dojo(object):
+    all_rooms = {}
+    dojo_persons = {}
+    all_office = []
+    all_ls = []
+    unallocated = {}
+
+    def create_room(self, room_type, room_name):
+        output = ""
+        for room in room_name:
+            if room in Dojo.all_rooms:
+                output = output + ('Room already exist.\n')
+            elif not room.isalnum():
```

Comment:
It's a Python inbuilt function. Maybe I will be adding a comment right on top of it. Thank you.